File: Level #1/Java/Izpit/src/_2SumCards.java
import java.util.*;
public class _2SumCards {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		String cards = input.nextLine();
		String[] card = cards.split("[SHDC] ");
		card[card.length - 1] = card[card.length - 1].substring(0, card[card.length - 1].length() - 1);
		int sum = 0;
		int mult = 1;
		int count = 1;
		
		for (int i = 0; i < card.length; i++) {
			if (card[i].equals(card[i+1])) {
				if (card[i].equals("2")) {
					sum += (2+2)*2;
				}
				 else if (card[i].equals("3")) {
					 sum += (3 + 3)*2;
				}
				 else if (card[i].equals("4")) {
					 sum += (4 + 4)*2;
					}
				 else if (card[i].equals("5")) {
					 sum += (5 + 5)*2;
					}
				 else if (card[i].equals("6")) {
					 sum += (6 + 6)*2;
					}
				 else if (card[i].equals("7")) {
					 sum += (7 + 7)*2;
					}
				 else if (card[i].equals("8")) {
					 sum += (8 + 8)*2;
					}
				 else if (card[i].equals("9")) {
					 sum += (9 + 9)*2;
					}
				 else if (card[i].equals("10")) {
					 sum += (10+ 10)*2;
					}
				 else if (card[i].equals("J")) {
					 sum += (12+12)*2;
					}
				 else if (card[i].equals("Q")) {
					 sum += (13+13)*2;
					}
				 else if (card[i].equals("K")) {
					 sum += (14+14)*2;
					}
				 else if (card[i].equals("A")) {
					 sum += (15+15)*2;
					}
			}
			else {
				if (card[i].equals("2")) {
					sum += 2;
				}
				 else if (card[i].equals("3")) {
					 sum += 3;
				}
				 else if (card[i].equals("4")) {
					 sum += 4;
					}
				 else if (card[i].equals("5")) {
					 sum += 5;
					}
				 else if (card[i].equals("6")) {
					 sum += 6;
					}
				 else if (card[i].equals("7")) {
					 sum += 7;
					}
				 else if (card[i].equals("8")) {
					 sum += 8;
					}
				 else if (card[i].equals("9")) {
					 sum += 9;
					}
				 else if (card[i].equals("10")) {
					 sum += 10;
					}
				 else if (card[i].equals("J")) {
					 sum += 12;
					}
				 else if (card[i].equals("Q")) {
					 sum += 13;
					}
				 else if (card[i].equals("K")) {
					 sum += 14;
					}
				 else if (card[i].equals("A")) {
					 sum += 15;
					}
			}
		}
		System.out.println(sum);
		
	}

}

File: Level #1/JavaScript/Exams/Exam4/1Юни изпит Java/2sumCards.js
function solve(input) {
    var cards = input[0].split(/[ CHDS]+/)
    cards.pop();
    var count = 1;
    var sum = 0;

    var cardsAll = [];
    for (var i in cards) {
        if(cards[i] == 'J'){
            cardsAll.push(parseInt(12))
        }
        else if(cards[i] == 'Q'){
            cardsAll.push(parseInt(13))
        }
        else if(cards[i] == 'K'){
            cardsAll.push(parseInt(14))
        }
        else if(cards[i] == 'A'){
            cardsAll.push(parseInt(15))
        }
        else{
            cardsAll.push(parseInt(cards[i]))
        }
    }

    for (var i = 0; i < cardsAll.length; i++) {
        while(cardsAll[i] == cardsAll[i + 1]){
            count++;
            i++;
        }
        if(count > 1){
            sum += cardsAll[i] * count * 2;
            count = 1;
        }
        else{
            sum += cardsAll[i];
        }
    }
    console.log(sum)
}
solve(['2C 2H 2D AS 10H 10C 2S KD']);
solve(['AS KH 10C']);
solve(['AS 10C KS KH KD 9H JH QS 3H QD QH 8S 10D 10S 7C JD']);